### ISSUE TITLE
where is the commands list in the official doc ?

### ISSUE THREAD

**Issue by jerome-diver** [tony/tmuxp#284]
Thank you for this tmuxp code.
Did you read the documentation ?
There is no list of usual command for use inside the yaml /json file.
Also, the doc and all the other copy/paste i read are terribly un-organised (sure there is not to much to know, but it is not a reason).
How people do for have the list of these functions ?
I would like to have the exhaustive list of functions i can use in the yaml (or json, anyway) file to load.
if you want, i should give you a formated (has you like) clear documentation with that (it is not so complicate also).
Do i have to find them somewhere in tmux doc ? or where ?
like this: 
[tmux doc / section variables names](http://man.openbsd.org/OpenBSD-current/man1/tmux.1)
but it seems to be a little bit different, for example, there is no "panes", so there is maybe not other ones...
that is confuse for me.
thank you

**Comment by tony** [comment]
Can you rephrase what you need help with, briefly?

The part of tmuxp that handles configuration files isn't the same as CLI commands.

YAML and JSON files are a declarative configuration. Sometimes the best way to express them is induction and example.

**Comment by jerome-diver** [comment]
yes @tony 
i'm searching for the list of ALL variables usable inside a yaml/json file.
(like window_name, panes, shell_command, etc... ALL of them).
Do you have that ?
or any link ?

**Comment by tony** [comment]
We can provide that, but I'm not sure the best way to convey it.

The other thing is, I haven't seen another project that uses YAML (like saltstack) find an appropriate way to *document* something that's kept in a hierarchical config. Which is why we've used examples this long.

So, properties can only be present in certain levels. For instance, "panes" would not work unless it was inside of "windows", e.g.

```yaml
panes:
- echo 1
```

wouldn't work.

It'd need to be inside a list of ``windows``, also there'd need to be a ``session_name``:

```yaml
session_name: minimal session
windows:
  - panes:
    - echo 1
```

Here's the json representation:

```js
{
  "windows": [
    {
      "panes": [
        "echo 1"
      ]
    }
  ],
  "session_name": "minimal session"
}
```

A bit of a wording clarification: we wouldn't refer to "windows", "panes", and "session_name" as variables. A better word would be properties.  Because there is already variable replacement inside the property values with [environmental variables](http://tmuxp.git-pull.com/en/latest/examples.html#environment-variable-replacing).
